### PR TITLE
Checkin

### PR DESCRIPTION
**Pull request by Aditiexcellence**

### CODE REVIEW

File: src/pages/Login/Login.vue [hunk 1]
```diff
@@ -135,6 +141,8 @@ export default {
     // }
   },
   created() {
+    // $cookies.set("cookie_name", "cookie_value");
```
Comment (vueexcel) [inline src/pages/Login/Login.vue]: remove non commented code

File: src/pages/Login/Login.vue [hunk 2]
```diff
@@ -74,7 +74,8 @@ export default {
   data() {
     return {
       loader: false,
-      loginfailed: false
+      loginfailed: false,
+      checkboxChecked: ""
```
Comment (vueexcel) [inline src/pages/Login/Login.vue]: use proper naming

File: src/pages/Checkin/Checkin.vue [hunk 3]
```diff
@@ -44,17 +48,26 @@
                       <br>
                       {{report.highlight}}
                     </p>
+                    <p @click="showData" v-if="canShowmore" class="readColor btn">Read more..</p>
```
Comment (vueexcel) [inline src/pages/Checkin/Checkin.vue]: use bootstrap class instead of using custom class

File: src/components/Area/Area.vue [hunk 4]
```diff
@@ -12,6 +12,13 @@
           </div>
         </div>
       </section>
+       <section v-else>
+        <div class="areaBorder">
```
Comment (vueexcel) [inline src/components/Area/Area.vue:16]: where the css define for this class?

File: src/Routes.js [hunk 5]
```diff
@@ -1,136 +1,159 @@
...
+      ]
+    }
+  ]
+});
+
+router.beforeEach((to, from, next) => {
+  if (to.matched.some(record => record.meta.requiresAuth)) {
+    // this route requires auth, check if logged in
+    // if not, redirect to login page.
+    if ($cookies.get("keepLoggedIn")) {
```
Comment (vueexcel) [inline src/Routes.js]: is this condition working as it should?

File: src/pages/Login/Login.vue [hunk 6]
```diff
@@ -97,8 +97,9 @@ export default {
         this.loader = true;
         this.api({ username: username, password: password })
           .then(resp => {
-            if (this.checkboxChecked !== "") {
+            if (this.signinChecked !== "") {
               $cookies.set("keepLoggedIn", this.authenticated);
+              console.log($cookies.get("keepLoggedIn"),'@@@@@');
```
Comment (vueexcel) [inline src/pages/Login/Login.vue]: always remove console before pushing

File: src/pages/WeeklyReview/WeeklyReview.vue [hunk 7]
```diff
@@ -168,20 +172,18 @@ export default {
...
-        { value: "6", text: "item6" },
-      ],
       kpiKraDescription: "",
       extraWorkDescription: ""
     };
   },
+  computed: {
+    user: get("profile/user")
+  },
+  created: function(){
```
Comment (vueexcel) [inline src/pages/WeeklyReview/WeeklyReview.vue]: why used JQuery?

File: src/pages/Login/Login.vue (hunk 1 above)
Comment (nodeexcel) [reply src/pages/Login/Login.vue]: will removed

File: src/pages/Login/Login.vue (hunk 2 above)
Comment (nodeexcel) [reply src/pages/Login/Login.vue]: naming has changed in next commit

File: src/pages/Checkin/Checkin.vue (hunk 3 above)
Comment (nodeexcel) [reply src/pages/Checkin/Checkin.vue]: bootstrap is used in next commit

File: src/components/Area/Area.vue (hunk 4 above)
Comment (nodeexcel) [reply src/components/Area/Area.vue:16]: in area.scss file

File: src/Routes.js (hunk 5 above)
Comment (nodeexcel) [reply src/Routes.js]: this is working in next commit

File: src/pages/Login/Login.vue (hunk 6 above)
Comment (nodeexcel) [reply src/pages/Login/Login.vue]: console removed in next commit

File: src/pages/WeeklyReview/WeeklyReview.vue (hunk 7 above)
Comment (nodeexcel) [reply src/pages/WeeklyReview/WeeklyReview.vue]: removed jquery in next commit

**Pull request merged**